### PR TITLE
Update French localization

### PR DESCRIPTION
**Pull request by surfdude29**
Following on from #4662, this PR adds translations for new strings and makes a tweak to an existing string.

I'd be grateful if you could take a look @Signez 🙏

### CODE REVIEW

File: src/locale/locales/fr/messages.po [hunk 1]
```diff
@@ -5030,6 +5049,10 @@ msgstr "Sélectionnez le(s) service(s) de modération destinataires du signaleme
 msgid "Select the service that hosts your data."
 msgstr "Sélectionnez le service qui héberge vos données."
 
+#: src/view/com/composer/videos/SelectVideoBtn.tsx:53
+msgid "Select video"
+msgstr "Sélectionne une vidéo"
```
Comment (Signez) [inline src/locale/locales/fr/messages.po:5054]: It's a label, so I would argue that it is, in fact, an infinitive we should use here.

```suggestion
msgstr "Sélectionner une vidéo"
```

**Review by Signez — COMMENTED** [review]
Great work as usual — and good catch of the missing `d’actu` (even though it wasn’t actually a problem, as it could have been understood thanks to the context).

File: src/locale/locales/fr/messages.po (hunk 1 above)
Comment (surfdude29) [reply src/locale/locales/fr/messages.po:5054]: Oops, yes that makes sense 👍

**Comment by surfdude29** [comment]
Correction applied, thanks yet again @Signez! :)

**Review by pfrazee — APPROVED** [review]
Thanks all!

**Pull request merged**